internal/consensus/common_test.go

```diff
@@ -397,6 +399,32 @@ func subscribeToVoter(ctx context.Context, t *testing.T, cs *State, addr []byte)
 	return ch
 }
 
+func subscribeToVoterBuffered(ctx context.Context, t *testing.T, cs *State, addr []byte) <-chan tmpubsub.Message {
```

Comment:
You could probably omit "Buffered" here, since by the time this merges to master, that distinction won't exist.